### PR TITLE
add missing coverage to sort methods

### PR DESCRIPTION
**Pull request by grosser**
@mcrossen

### CODE REVIEW

**Comment by grosser** [comment]
@mcrossen 👋

File: lib/deepsort.rb [hunk 1]
```diff
@@ -86,7 +86,7 @@ def deep_sort_by!(&block)
           value
         end]
 
-      end.sort_by(&block)])
+      end.sort_by!(&block)])
```
Comment (mcrossen) [inline lib/deepsort.rb:89]: isn't the in-place operator covered by the closing replace() here?

**Comment by mcrossen** [comment]
it looks like the travis status shows up on pull requests again. that's nice.

just address my question please and I'll merge this.

File: lib/deepsort.rb (hunk 1 above)
Comment (grosser) [reply lib/deepsort.rb:89]: yes, but this way we produce 1 less throwaway object

**Pull request merged**